packages/core/src/App/Containers/app-notification-messages.jsx

```diff
@@ -19,23 +20,36 @@ class AppNotificationMessages extends React.Component {
     };
 
     render() {
-        const allowed_on_mobile = ['unwelcome', 'contract_sold'];
+        const allowed_on_mobile = ['unwelcome', 'contract_sold', 'dp2p'];
 
-        const { marked_notifications, notification_messages, removeNotificationMessage } = this.props;
+        const { is_p2p_visible, marked_notifications, notification_messages, removeNotificationMessage } = this.props;
```

Comment:
Same as above, platform specific exceptions in core should be avoided if possible.